src/stringTemplates/desktop-distribution.tpl

```diff
@@ -1,7 +1,3 @@
--- Please ensure that you have selected a BUILD ID (by clicking
--- on the desired reference point) before proceeding with this
--- template in STMO.
-
 -- For more info on the tables referenced in this SQL see:
 -- https://docs.telemetry.mozilla.org/datasets/main_ping_tables.html
```

Comment:
I'll open a separate PR in data docs to update instruction on how to create distribution view.